src/Compilers/VisualBasic/Portable/Symbols/Source/SourceNamedTypeSymbol.vb

```diff
@@ -99,6 +100,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
 #Region "Completion"
         Protected Overrides Sub GenerateAllDeclarationErrorsImpl(cancellationToken As CancellationToken)
+#If DEBUG Then
+            EnsureAllHandlesAreBound()
+#End If
```

Comment:
I should probably set ```_withEventsOverridesAreFrozen``` to true after ```MyBase.GenerateAllDeclarationErrorsImpl``` so that EnsureAllHandlesAreBound could take advantage of that fact.